resources/assets/components/StatusFilter/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+class TagsFilter extends React.Component {
```

Comment:
It looks like you are calling this `TagsFilter` in a `StatusFilter` component?

Reply:
We actually should probably have this be `DropdownFilter` that takes in props that define what the `options` should be.

Reply:
oops!